pheget/api/__init__.py

```diff
@@ -2,13 +2,73 @@
 API endpoints (return JSON, not HTML)
 """
 
+import json
+import sqlite3
+import urllib.request
+
 from flask import Blueprint, abort, jsonify, request
 
+from .. import model
 from .format import query_variants
 
 api_blueprint = Blueprint("api", __name__)
 
 
+@api_blueprint.route("/bestvar/<string:symbol>/", methods=["GET"])
```

Comment:
As a site grows, sometimes it can get hard to remember what a route as called, or the syntax it uses.

One way to keep this organized is to apply common rules throughout, and treat relationships as nested routes. (so everything about a single variant is somewhere under `/variant/`)

Would it make sense to call this something like the following?

`/variant/<string:chrom>_<int:pos>/best/`

If this refers to a gene instead of a variant: we can imagine other gene data in this site in the future. Maybe then the top level entity would be `/gene/symbol/bestvar/`, etc

Note the child URL, and setting consistent syntax for how a variant is defined across both routes.

Reply:
Yes, this refers to a gene (the query is either a gene name or gene_id, and it returns the chr:pos of the variant with the best eQTL signal), so it can be treated as a property of a gene.

`/gene/bestvar/` might work, since it should return the same result if you put in either a short gene name (e.g. "CHRNA5") or the longer gene_id ("ENSG00000169684").

Reply:
I've changed it to /api/<string:symbol>/bestvar/, which I hope makes sense: it returns the "best variant" for a given gene (symbol or ENSG...)